package.json

```diff
@@ -0,0 +1,38 @@
+{
+  "name": "custom-html",
+  "version": "1.0.0",
+  "description": "A Managed Component that injects HTML on your page",
+  "main": "dist/index.js",
+  "scripts": {
+    "lint": "eslint --ext .ts,.js, src",
+    "lint:fix": "eslint --ext .ts,.js, src --fix",
+    "bundle": "node esbuild.js",
+    "build": "npm run test && npm run lint && npm run typecheck && npm run bundle",
+    "typecheck": "tsc --project tsconfig.build.json --noEmit",
+    "test": "vitest run --globals",
+    "test:dev": "vitest --globals"
```

Comment:
If we can also add a
`"release": "npm run build && npm version patch && npm publish"`
This should ideally be in the MC template but isn't :/